packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -178,11 +178,14 @@ contract LockedGold is
     Balances storage account = balances[msg.sender];
     // Prevent unlocking gold when voting on governance proposals so that the gold cannot be
     // used to vote more than once.
-    require(!getGovernance().isVoting(msg.sender), "Account locked");
+    uint256 remainingLockedGold = getAccountTotalLockedGold(msg.sender).sub(value);
+
+    uint256 totalReferendumVotes = getGovernance().getAmountOfGoldUsedForVoting(msg.sender);
```

Comment:
Doesn't this approach duplicate vote counting?
Like if I vote for proposal A with 1 CELO and for proposal B with 1 CELO, then IIUC this function will return 2, but in reality I'm using 1 CELO and voting twice with it. So I should be able to unlock everything but 1 CELO.

So `getAmountOfGoldUsedForVoting` should return the max rather than the count (or maybe create another function as `getAmountOfGoldUsedForVoting` may be useful for something else)

Reply:
getAmountOfGoldUsedForVoting is currently returning max of gold used for voting. Please check Governance.sol:1417